src/test/RUNTESTS

```diff
@@ -127,6 +127,7 @@ runtest() {
 				[ $retval = 124 -o $retval = 137 ] && errmsg='timed out'
 
 				[ $retval != 0 ] && {
+					[ -t 1 ] && errmsg="\033[31m$errmsg\033[m"
```

Comment:
I like it, but when I was growing up there were so many different types of terminals that you would never dream of hard-wiring ANSI sequences into a script.  Maybe times have changed :-)

Any reason not to use `tput setaf 1` and `tput sgr0` instead of the hard-wired sequences?